frontend/src/utils/role.ts

```diff
@@ -44,6 +45,7 @@ export const WORKSPACE_PERMISSION_MATRIX: Map<
   ["bb.permission.workspace.manage-member", [false, false, true]],
   ["bb.permission.workspace.manage-vcs-provider", [false, false, true]],
   ["bb.permission.workspace.manage-general", [false, false, true]],
+  ["bb.permission.workspace.manage-watermark", [false, true, true]],
```

Comment:
We can reuse bb.permission.workspace.manage-general which is used for configuring branding

Reply:
SGTM

Reply:
updated.